src/type/schema.js

```diff
@@ -32,6 +32,7 @@ import {
   type GraphQLNamedType,
   type GraphQLAbstractType,
   type GraphQLObjectType,
```

Comment:
You forget to synchronize this file with `schema.d.ts`

Reply:
Good catch!